src/Altinn.App.Core/Models/Layout/Components/_readme.md

```
@@ -1,3 +1,11 @@
-Most components are handled in C# with the generic `Altinn.App.Core.Models.Layout.Components.BaseComponent`
-that includes the most common fields in components. Some component types have other fields that have meaning in backend
-(or is used in backend validation) thus a subclass of `BaseComponent` is used.
\ No newline at end of file
+## Working with layouts in C#
```

markdownlint-cli2 (0.17.2)

No space inside hashes on closed atx style heading

(MD020, no-missing-space-closed-atx)